test/unit/basic_test.cpp

```diff
@@ -27,13 +27,15 @@
  ****************************************************************************/
 
 #include "gtest_include.h"
+#include <proj.h>
 
 namespace {
 
 // ---------------------------------------------------------------------------
 
 TEST( dumy, dummy ) {
-    ASSERT_EQ( 1 , 1 );
+    PJ* p = ::proj_create(PJ_DEFAULT_CTX, "+proj=merc +ellps=clrk66 +lat_ts=33");
+    ASSERT_NE(p, nullptr);
```

Comment:
you need a ```::proj_destroy(p);``` to make leak sanitizer happy :-)

Reply:
I blame the `dummy` name :-)
It should be fixed now.
For serious tests, that's gonna be taken care of by fixture(s).